## 21manpage_nethack_activation.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Document that there are two possible causes for 'nethack on'
## DP: being activated by default...

@DPATCH@
diff -Naur screen-4.0.3.orig/doc/screen.1 screen-4.0.3/doc/screen.1
--- screen-4.0.3.orig/doc/screen.1	2008-06-01 23:44:57.000000000 +0200
+++ screen-4.0.3/doc/screen.1	2008-06-01 23:44:58.000000000 +0200
@@ -2268,7 +2268,8 @@
 .I screen
 was compiled with the NETHACK flag defined. The
 default setting is then determined by the presence of the environment 
-variable $NETHACKOPTIONS.
+variable $NETHACKOPTIONS and the file ~/.nethackrc - if either one is present,
+the default is \fBon\fP.
 .sp
 .ne 3
 .B next
diff -Naur screen-4.0.3.orig/doc/screen.texinfo screen-4.0.3/doc/screen.texinfo
--- screen-4.0.3.orig/doc/screen.texinfo	2008-06-01 23:44:57.000000000 +0200
+++ screen-4.0.3/doc/screen.texinfo	2008-06-01 23:44:58.000000000 +0200
@@ -4447,7 +4447,8 @@
 This option is only available if @code{screen} was compiled with the
 NETHACK flag defined (@pxref{Installation}). The default setting is then
 determined by the presence of the environment variable
-@code{$NETHACKOPTIONS}.
+@code{$NETHACKOPTIONS} and the file @code{~/.nethackrc} - if either one is
+present, the default is @code{on}.
 @end deffn
 
 @node Nonblock, Number, Nethack, Miscellaneous
